src/providers/url/url.ts

```diff
@@ -15,7 +15,7 @@ export class UrlProvider {
   }
 
   getLoggingServiceUrl(): any {
-    return this.appConfigProvider.getAppConfig().loggingUrl;
+    return this.appConfigProvider.getAppConfig().logging.url;
```

Comment:
What are your thoughts about changing this to getLogsServiceUrl . It seems like everything else has moved from logging to logs

Reply:
Also the microservice is called `mes-logs-service` so ye it totally makes sense. But that would require an additional PR in the configuration microservice. Do you think that it's worth it?

Reply:
Changed it and refactored the code so that we use `logs` everywhere (Global hah).